components/IdentifyViewer.jsx

```diff
@@ -226,6 +226,19 @@ class IdentifyViewer extends React.Component {
         if(this.state.exportFormat === 'json') {
             let data = JSON.stringify(json, null, ' ');
             FileSaver.saveAs(new Blob([data], {type: "text/plain;charset=utf-8"}), "results.json");
+        } else if(this.state.exportFormat === 'geojson') {
+            Object.entries(json).forEach(([layerName, features]) => {
+                let data = {
+                    "type":"FeatureCollection",
+                    "features":[]
```

Comment:
Why not just

    "features": features

?

Reply:
I tried to put` "features": feature` but this gives me `[object Object]` in my geojson files even if I put `JSON.stringify` after `let data`. Also I need to look over all the features to not have the layer's name in the geojson.

Reply:
You are already separating features from layernames by iterating over `Object.entries(json)`. This

    features.forEach(feature => {
                    data.features = data.features.concat(feature);
    })

is really the same as `data.features = features`. And you also shouldn't need to reserialize the data in that loop.

Reply:
Sorry, I didn't understand at the first comment but now I changed the code. Thanks for your advices.